server/main.go

```diff
@@ -7,36 +7,71 @@ import (
 	"net/http"
 	"os"
 	"time"
+
+	"github.com/prometheus/client_golang/prometheus"
 )
 
 const (
-	exitFail = 1
+	exitFail      = 1
+	serverTimeout = 3
+	tickerTime    = 100
```

Comment:
You should be using durations here or call it `tickerSeconds` to make sure the unit is clear.